aspnetcore/signalr/hubs.md

```diff
@@ -129,6 +129,8 @@ Override the `OnDisconnectedAsync` virtual method to perform actions when a clie
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/signalr/hubs/samples/6.x/SignalRHubsSample/Snippets/Hubs/ChatHub.cs" id="snippet_OnDisconnectedAsync":::
```

Comment:
@wadepickett  per my request, update all snippets, including this one.

Reply:
I will set this to draft to save confusion.  Not done yet.  Yes Rick, I need to work through the snippets.  You may see several commits.   I did not miss your request.